en/docs/api-management/api-policies.md

```diff
@@ -42,6 +42,18 @@ To attach a policy to the `Request`, `Response`, or `Error` flow of a REST API p
 
 8. Click **Save**.
 
+After attaching an API Policy, it is necessary to deploy the API in order for the policy to become active within its corresponding flow. Some of the default policies in Choreo require you to specify custom parameters and values. If the policy you attached requires environment-specific or custom values for its parameters, you can indicate them during the API deployment process.
```

Comment:
`If the policy you attached requires environment-specific or custom values for its parameters,`

for this section, shall we point to the **Configure a policy to accept environment-specific parameters** topic?